staff/ana-sanjuan/skylab-precourse-projects/README.md

```diff
@@ -11,16 +11,16 @@ I hope you enjoy them!
 ## Projects
 
```

Comment:
this file SHOULD NOT be included in this PR. you have mixed two branches into develop or you have been working both things in the same branch. please, take care and do not mix work again.

Reply:
Yes, I merged two branches into develop, and I am afraid I did it intentionally. I thought it was the only way to update the projects if there was already an open pull request in develop... I see I was wrong, next time will ask before! thanks